documentation/concepts.md

```diff
@@ -84,13 +122,19 @@ To accommodate this, streams are partitioned. By default, a stream consists of
 just a single partition, but multiple partitions can be created for increased
 parallelism. Messages can then be delivered to partitions based on their key,
 in a round-robin fashion, randomly, or with some other partitioning strategy
-on the client.
+on the client. 
 
+```
+Use case note:
+Please note that ordering within the partition is upheld but not across partitions. This means that the partitioning strategy is of high importance, since an aggregated state cannot be achieved for a path dependent consumer subscribing to many streams (but perhaps on the same NATS subject). I.e. a consumer needing correct order of events can only subscribe to many streams if and only if the events are uncorrelated/independent between the partitions (and thus have order within a partition.) An architect would pay particular interest in making sure of independent and stateless workers when applying DDD. Link of interest: https://dddcommunity.org/book/evans_2003/
+```
 Additionally, streams can join a named load-balance group, which load balances
 messages on a NATS subject amongst the streams in the group. Load-balance
 groups do not affect message delivery to other streams not participating in
 the group.
 
+[@Tyler - pls descibe use case or advise set up on architecture impementatiob for user]
```

Comment:
Load-balance groups are for distributing messages from a subject amongst a group of streams. Note that this could also be achieved using a _partitioned_ stream with a random or round-robin partitioning strategy. However, partitioning assumes you've already "bought in" to Liftbridge streams as a first-class citizen. But imagine a case where there is a pre-existing NATS subject that Liftbridge-agnostic services are already publishing to, and we want to turn that NATS subject into a durable log that Liftbridge-aware services can consume. We could attach a single stream to the subject, but if it's very high volume, we might need a load-balance group to distribute the load across a set of streams.